homeassistant/components/yolink/sensor.py

```diff
@@ -52,6 +52,7 @@ class YoLinkSensorEntityDescription(
 
 SENSOR_DEVICE_TYPE = [
     ATTR_DEVICE_DOOR_SENSOR,
+    ATTR_DEVICE_LEAK_SENSOR,
```

Comment:
Is this a fix? Is so, please break it out to a separate PR.

Reply:
Yes, but it's not a very urgent fix， Should i create a new PR for this?

Reply:
Yes, please. It's better to always separate fixes from feature additions and only do a single thing in each PR. Thanks!

Reply:
Ok, I will do this